apps/designer/app/designer/features/sidebar-left/panels/components/image.tsx

```diff
@@ -67,6 +81,37 @@ export const AssetManagerImage = ({
           ...(isUploading ? { filter: "blur(1px)", opacity: 0.7 } : {}),
         }}
       ></Box>
+      {!isUploading && (
+        <Box ref={tooltipRef}>
+          <Tooltip
```

Comment:
is this potentially how our default tooltip inside design system should always be styled?

Reply:
yeah, I agree, gonna change it in the tooltip